.github/workflows/create-release.yml

```diff
@@ -1,19 +1,137 @@
-name: 'tagged-release'
+name: 'Official Release'
```

Comment:
delete this file now?

Reply:
semantic release config determines what kind of release on what branch, this shouldnt need to be run